apps/sms/test/unit/thread_list_ui_test.js

```diff
@@ -173,16 +163,6 @@ suite('thread_list_ui', function() {
       MockOptionMenu.mTeardown();
     });
 
-    test('show settings/cancel options when list is empty', function() {
-      ThreadListUI.setEmpty(true);
-      ThreadListUI.showOptions();
-
-      var optionItems = MockOptionMenu.calls[0].items;
-      assert.equal(optionItems.length, 2);
-      assert.equal(optionItems[0].l10nId, 'settings');
-      assert.equal(optionItems[1].l10nId, 'cancel');
-    });
-
```

Comment:
no you can't really do that because we don't apply the CSS files in unit tests.

You can try doing a integration tests, but it's not really easy so I won't force you to do one.